rbac/migration_tool/sharedSystemRolesReplicatedRoleBindings.py

```
@@ -166,10 +166,13 @@ def permission_groupings_to_v2_role_bindings(
     perm_groupings: PermissionGroupings, v1_role: Role, role_bindings: Iterable[BindingMapping]
 ) -> list[BindingMapping]:
     """Determine updated role bindings based on latest resource-permission state and current role bindings."""
-    updated_mappings: list[BindingMapping] = []
-    latest_roles_by_id: dict[str, V2role] = {}
     # TODO: this is broken for system roles, need to have Tenant or Policies provided
     # so that we don't look up Policies across all Tenants!
+    if v1_role.system:
+        raise NotImplementedError("System roles are not supported.")
```

I think we should also raise `ValueError` here instead like others since its not missing functionality :)